modules/ami2/nomad-consul-docker-ecr.json

```diff
@@ -4,9 +4,7 @@
     "nomad_version": "0.9.0",
     "consul_module_version": "v0.4.4",
     "consul_version": "1.4.4",
-    "aws_account_ids": "",
-    "aws_region": "eu-central-1",
```

Comment:
Configuring the region information in packer turned out to be an antipattern, because this way packer can not be used in Terratest to create temporary AMIs.
From my perspective it is better to configure the region settings as command line parameter, this way it is flexible and explicit.

Reply:
Ok for me